api_end_end_testing/src/test/java/tds/user/api/tests/BaseUri.java

```diff
@@ -0,0 +1,31 @@
+package tds.user.api.tests;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import org.testng.annotations.BeforeClass;
+
+/**
+ * Created by mjbarger on 9/6/16.
+ */
+public abstract class BaseUri {
+    String authenticateURI = null;
+    String userUri = null;
+
+    @BeforeClass
+    public void init() {
+
+        Properties prop = new Properties();
+        InputStream inputStream = getClass().getClassLoader().getResourceAsStream("config.properties");
+
+        try {
+            prop.load(inputStream);
+            System.out.println("auth prop " + prop.getProperty("authenticateURI"));
+        }
+        catch (IOException e) {
+            e.printStackTrace(System.out);
```

Comment:
Rather than printing out a stacktrace what if instead we did `fail(e.getMessage())`?  That way the test run would just end right then.  And if that doesn't work I suggest doing going with `throw new RuntimeException(e)`.  Either way that would result in the test catastrophically failing which is what I think we want since the tests will all fail anyway without the properties file.